### DISCUSSION TITLE
Sorting Files in Nested Folders

### DISCUSSION THREAD

**Discussion by Xaerwyn** [SebastianMC/obsidian-custom-sort#43] [Q&A] [answered]
Hey I can't seem to custom sort files that are inside a folder that is inside another folder. Is this not possible or am I just missing something? Haven't had any problems sorting files that are only inside 1 folder.

Thanks!

**Comment by SebastianMC** [comment, marked as answer]
Hi @Xaerwyn 

not having more details about your case I can only guess

My most obvious suspicion is the syntax to specify the target folder.\
There are two variants supported:
- `/...` which only applies to the specified folder and its immediate subfolders and not deeper, AND
- `/*` which applies to the specified folder and the entire hierarchy of subfolders. In other words, all descendant subfolders (sub-subfolders, and so on)

And for example, respectively:
- `target-folder: MyNotes/...` -> will apply the specified sorting rule to the folder `MyNotes` and its immediate children like `MyNotes/2020` and `MyNotes/2021`. And **will NOT apply** the sorting rule to more nested folders like `MyNotes/2020/Jan` or `MyNotes/2021/Attachment/images`
- ```target-folder: MyNotes/*``` -> will apply the sorting rule to the folder and its all descendants folders, any depth, including examples like `MyNotes/2020`, `MyNotes/2021`, `MyNotes/2020/Jan`, `MyNotes/2021/Attachment/images`

So, the obvious guess, based on your description would be to switch from the `/...` suffix in target folder spec to `/*` suffix.

Let me know if that is or is not the case.

**Reply by Xaerwyn** [reply]
Ah yes that worked thank you!
